apps/showdoc/3.3.0/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   showdoc:
-    image: star7th/showdoc:v3.3.0
+    image: star7th/showdoc:v3.3.1
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
It appears that there's no need to change this code at the moment it is presented. It looks fine but could potentially be updated with more recent documentation information should an update happen for ShowDoc version v3.3.1 before 2024-12-07.